extensions/vscode/src/types/messages/webviewToHostMessages.ts

```diff
@@ -14,13 +14,15 @@ export enum WebviewToHostMessageType {
   INCLUDE_FILE = "includeFile",
   EXCLUDE_FILE = "excludeFile",
   REQUEST_FILES_LISTS = "requestFilesLists",
+  REQUEST_CREDENTIALS = "requestCredentials",
   VSCODE_OPEN_RELATIVE = "VSCodeOpenRelativeMsg",
   REFRESH_PYTHON_PACKAGES = "RefreshPythonPackagesMsg",
   SCAN_PYTHON_PACKAGE_REQUIREMENTS = "ScanPythonPackageRequirementsMsg",
   REFRESH_R_PACKAGES = "RefreshRPackagesMsg",
   SCAN_R_PACKAGE_REQUIREMENTS = "ScanRPackageRequirementsMsg",
   SELECT_DEPLOYMENT = "selectDeployment",
   NEW_DEPLOYMENT = "newDeployment",
+  NEW_CREDENTIAL_FOR_DEPLOYMENT = "newCredentialForDeployment",
```

Comment:
I changed the name here since this used the selected deployment, and we wanted to make a new "plain" credential without the URL pre-filled. `NEW_CREDENTIAL` seemed a better name for that so made the switch.